docs/validators/get-started/using-cosmovisor.md

```diff
@@ -5,14 +5,14 @@
 ## Installation
 
 ::: tip Cosmovisor Versions
-The following instructions use the latest version (`v1.2`), which does not support installation with `go install` and therefore `cosmovisor` must be built from source.
-
-The most recent version that supports `go install` is `v1.0`, which is still safe to use but the configuration will be different and several improvements have since been added.
+The following instructions use the latest version (`v1.4`), which now supports installation with `go install`.
 
 For more information about each version, see the release notes:
 - [Cosmovisor v1.0.0](https://github.com/cosmos/cosmos-sdk/releases/tag/cosmovisor%2Fv1.0.0)
 - [Cosmovisor v1.1.0](https://github.com/cosmos/cosmos-sdk/releases/tag/cosmovisor%2Fv1.1.0)
 - [Cosmovisor v1.2.0](https://github.com/cosmos/cosmos-sdk/releases/tag/cosmovisor%2Fv1.2.0)
+- [Cosmovisor v1.3.0](https://github.com/cosmos/cosmos-sdk/releases/tag/cosmovisor%2Fv1.3.0)
+- [Cosmovisor v1.4.0](https://github.com/cosmos/cosmos-sdk/releases/tag/cosmovisor%2Fv1.4.0)
 :::
 
 Clone the `cosmos-sdk` repository (if not already cloned):
```

Comment:
I think we can actually reframe this whole section and start by telling ppl how to use `go install` since its a specific path they need to run:

```
go install cosmossdk.io/tools/cosmovisor/cmd/cosmovisor@v1.4.0
```

As described in https://github.com/cosmos/cosmos-sdk/releases/tag/tools%2Fcosmovisor%2Fv1.4.0


And then the rest should all be labelled as an alternative to `go install` where you build from source.